src/App.css

```diff
@@ -5,16 +5,25 @@
 
 body {
   padding: 24px;
+  background-color: #f5f6fa;
 }
 
 main {
-  margin: 12px 0;
+  /* margin: 12px 0;
+  background-color: #fafafa; */
```

Comment:
- [ ] kindly consider removing all unused commented code, it makes your code messy and cluttered